packages/dashboard/README.md

```diff
@@ -8,109 +8,26 @@ Follow the build instructions [here](../../README.md/#launching).
 
 ## Configuration
 
-### (Optional) Import external resources.
+Before building the dashboard, you must configure it according to the deployment. All the configurations is on `app-config.json`, the default configuration is for running the api-server and dashboard locally. You may check [src/app-config.ts](src/app-config.ts) for more details on the config file schema.
 
-This allows for build-time configurations of the dashboard.
+### (Optional) Add external resources.
 
-For dashboard resources for `rmf_demos` simulation worlds, check out the `rmf-web-dashboard-resources` branch of `rmf_demos`.
```

Comment:
This part of the README is important and should not be left out, we will need to update the demos resources to fit this new schema.

edit: perhaps adding below https://github.com/open-rmf/rmf-web/pull/977/files#diff-dd8694e64a96330e60855260f8dca122283229b1ba3d457ce6e498e3778e0c23R28 where you describe `/resources/defaultLogo.png`

Reply:
The problem is that it is still using the old resource manager pattern. I guess I could mention it here and update the demo resources later.

Reply:
yeah, or if we point directly to github for the icons, it may be indicative enough. Probably a short snippet will be useful still, mostly just knowing where the icons are would be useful